### PR TITLE
add texture maps

### PR DESCRIPTION
**Pull request by alpcihan**

### CODE REVIEW

**Comment by alpcihan** [comment]
[Support different materials + textures](https://trello.com/c/BPHOCEIz/20-support-different-materials-textures)

File: lib/core/scene/scene-data-manager.ts [hunk 1]
```diff
@@ -110,7 +127,13 @@ class SceneDataManager {
     private m_blasArray: Array<BLAS>;
     private m_blasNodeCount: number; // total node count
     private m_blasInstanceArray: Array<BLASInstance>;
+
     private m_materials: Array<Material>;
+    private m_totalMapCount: number;
+    private m_albedoMapCount: number;
```
Comment (umuterc) [inline lib/core/scene/scene-data-manager.ts]: map counts are not used. Are we using them somewhere else?

Comment (alpcihan) [reply lib/core/scene/scene-data-manager.ts]: forgot to delete them. will delete them now.

File: lib/loaders/gltf-loader.ts [hunk 2]
```diff
@@ -91,7 +91,7 @@ class WTGLTFLoader {
 
             // load material textures
             if (threeMat.map !== null) {
-                material.albedoTexture = new Texture(threeMat.map.image);
+                material.albedoMap = new Texture(threeMat.map.image);
                 material.baseColor = new THREE.Vector3(-1, -1, -1);
```
Comment (umuterc) [inline lib/loaders/gltf-loader.ts]: We can remove this line.

Comment (alpcihan) [reply lib/loaders/gltf-loader.ts]: material.baseColor .... should be removed already

**Review by umuterc — APPROVED** [review]

**Pull request merged**